.gitignore

```diff
@@ -0,0 +1,3 @@
+/node_modules
+/.idea
+package-lock.json
```

Comment:
Remove this file from gitignore. We should commit this file
Can you also add .nvmrc and .npmrc files as well?

Reply:
Yep. Added that by mistake.